Makefile

```diff
@@ -10,7 +10,7 @@ run: setup
 
 # Run local tests
 test: setup
-	cd pytest && poetry run pytest -n 8 --setup-show selftests/
+	cd pytest && poetry run pytest --keep-vms --targets=targets.yaml --setup-show selftests/
```

Comment:
Is this one used for local testing? Cannot find the targets.yaml file sample, not able to run a local test.

Reply:
Yes, in testing. Sorry, I must not have pushed it. Adding now.